### ISSUE TITLE
Specify all arguments in the examples of the package documentation

### ISSUE THREAD

**Issue by stefan-mueller** [quanteda/quanteda#1510]
Many man pages in the package documentation do not explicitly specify some of the arguments (e.g. `measure`/`dictionary`/`pattern`). I could go through the documentation and add them to make the examples more robust and comprehensible.

**Comment by kbenoit** [comment]
Not sure what you mean here, since all man pages must specify arguments in a function. Can be give and more detailed example of the problem?

**Comment by stefan-mueller** [comment]
Yes, sorry for the confusion – I only meant the argument names. Since I already changed and extended the documentation in #1504, I could specify the argument names in that branch (`revise-documentation`) to avoid conflicts when merging?

**Comment by kbenoit** [comment]
I'm working on the fix for #1506 so I can merge that branch now, should be done shortly, so let's get that one fixed first.
